scripts/programming_files.py

```diff
@@ -417,7 +451,33 @@ def write_d878_zip(df: pd.DataFrame) -> None:
     # D878 CPS software requires CR/LF line endings
     df.to_csv("assets/programming_files/d878.csv", lineterminator="\r\n")
 
-    # TODO : generate scanlist and talk groups files here
+    # Generate a scan list for each region
+    scan_lists = {region: df[df["Scan List"] == region] for region in df["Scan List"].unique()}
```

Comment:
```suggestion
    scan_lists = {region: df.loc[df["Scan List"] == region] for region in df["Scan List"].unique()}
```

I'm suggesting this change because it's very minorly more explicit. A little background [in this older thread](https://stackoverflow.com/questions/53297140/pandas-loc-vs-non-loc-for-boolean-indexing).

Mostly there's a conflict with the d878.zip so you'll need to `python scripts/update.py --regen` anyway so I thought I might as well give you a reason to ;)